Assets/Scripts/SaberToothedTiger.cs

```diff
@@ -0,0 +1,65 @@
+using Controllers;
+using Props;
+using UnityEngine;
+
+public class SaberToothedTiger : Smashable
+{
+    private Dodo _dodoToChase;
+
+    // Animation
+    private Animator _anim;
+    private static readonly int RunningSpeed = Animator.StringToHash("runningSpeed");
+
+    // Speed and movement values
+    [Tooltip("This is how much faster the tiger is than the world (at time of spawn).")] [SerializeField]
+    private float speedAboveWorld;
+
+    private float _fullSaberSpeed;
+    private Vector3 _directionOfFocus;
+
+    [SerializeField] private float sidewaysSpeed = 2f;
+
+    //Vector line that Tiger will moves on TODO this is wrong because it depends where the tiger is on the screen...
+    private Vector3 _sideVector3 = new Vector3(.5f, 0.25f) / 100;
+    [SerializeField] Vector3 tigerForwardVector = new Vector3(1f, -.5f);
+
+
+    // Start is called before the first frame update
+    protected override void Start()
+    {
+        _worldController = FindObjectOfType<WorldController>();
+        // On spawn we want the tiger to start running towards the dodo.
+        _fullSaberSpeed = _worldController.getWorldSpeed() + speedAboveWorld;
+        _dodoToChase = FindObjectOfType<Dodo>();
+        _anim = GetComponent<Animator>();
+        _anim.SetFloat(RunningSpeed,
+            speedAboveWorld + _worldController.getWorldSpeed() / _worldController.getWorldSpeedPercentage() / 2.5f);
+    }
+
+    // Update is called once per frame
+    protected override void Update()
+    {
+        MoveTowardsDodo();
+        Vector3 movementVector = _worldController.getMoveDirection() *
+                                 (Time.deltaTime * (_fullSaberSpeed - _worldController.getWorldSpeed()));
+        transform.position += movementVector;
+    }
+
+    private void MoveTowardsDodo()
+    {
+        Vector3 currentPos = transform.position;
+        _directionOfFocus = currentPos - _dodoToChase.transform.position;
+
+        //Is smelled object to Dodo's left or right
```

Comment:
Ima guess this is copy pasted from some dodo logic ;)

Reply:
NEVER
it's correct naming though :D